packages/now-cli/src/util/dev/server.ts

```diff
@@ -1952,7 +1954,11 @@ export default class DevServer {
       `Running Dev Command ${chalk.cyan.bold(`“${devCommand}”`)}`
     );
 
-    const port = await getPort();
+    let port = await getPort();
+
+    if (this.frameworkSlug === 'redwoodjs') {
+      port = await getRedwoodPort(cwd);
+    }
```

Comment:
@TooTallNate I'm open to ideas if you can think of a better way to handle this besides special case.

I created a PR upstream but it doesn't seem like we'll get it soon https://github.com/redwoodjs/redwood/pull/891

Reply:
I think it's fine considering that it's just temporary until upstream provides the proper functionality.